### ISSUE TITLE
empty (p/promise) just returns nil

### ISSUE THREAD

**Issue by mpenet** [funcool/promesa#61]
Looking at https://github.com/funcool/promesa/blob/master/src/promesa/impl.cljc#L183 it seems that the :clj branch just returns nothing. I guess you meat it to return something like `(java.util.concurrent.CompletableFuture.)` ?

**Comment by niwinz** [comment]
Seems like a bug.

**Comment by mpenet** [comment]
You prolly need to add all of the operations support too: 
ex: 
```
(extend-protocol pt/ICompletable
  java.util.concurrent.CompletableFuture
  (-resolve [f v] (.complete f v))
  (-reject [f v] (.completeExceptionally f v)))
```

And the same for cancelable & co I think.

**Comment by niwinz** [comment]
Should be fixed in 2.0.1
